lib/pfconfig/namespaces/config/DNS_Filters.pm

```diff
@@ -18,14 +18,16 @@ use strict;
 use warnings;
 
 use pfconfig::namespaces::config;
-use pf::file_paths qw($dns_filters_config_file);
+use pf::file_paths qw($dns_filters_config_file $dns_filters_default_config_file);
 
 use base 'pfconfig::namespaces::config';
 
 sub init {
     my ($self) = @_;
     $self->{file} = $dns_filters_config_file;
     $self->{child_resources} = [ 'FilterEngine::DNS_Scopes'];
+    my $defaults = Config::IniFiles->new( -file => $dns_filters_default_config_file);
+    $self->{added_params}->{'-import'} = $defaults;
```

Comment:
Actually both styles are used.
I would prefer two separate lines (surprise!!!).
For the case where we would want to look at the defaults content for debugging.

If we are going to standardize to a style I think it is best to standardize to the one that would make it easier to debug in the future.


```

ag added_params lib/pfconfig/

lib/pfconfig/namespaces/config/Profiles.pm
43:    $self->{added_params}{'-import'} = Config::IniFiles->new(-file => $profiles_default_config_file);

lib/pfconfig/namespaces/config/Switch.pm
36:    $self->{added_params}{'-import'} = Config::IniFiles->new(-file => $switches_default_config_file);

lib/pfconfig/namespaces/config/Realm.pm
34:    $self->{added_params}->{'-import'} = $defaults;

lib/pfconfig/namespaces/config/Violations.pm
31:    $self->{added_params}{'-import'} = Config::IniFiles->new(-file => $violations_default_config_file);

lib/pfconfig/namespaces/config/PortalModules.pm
34:    $self->{added_params}->{'-import'} = $defaults;

lib/pfconfig/namespaces/config/Pfqueue.pm
38:    $self->{added_params}{'-import'} = Config::IniFiles->new(-file => $pfqueue_default_config_file);

lib/pfconfig/namespaces/config/VlanFilters.pm
34:    $self->{added_params}->{'-import'} = $defaults;

lib/pfconfig/namespaces/config/DNS_Filters.pm
30:    $self->{added_params}->{'-import'} = $defaults;

lib/pfconfig/namespaces/config.pm
39:    $self->{added_params}      = {};
55:    $self->{added_params}->{-file} = $self->{file};
56:    $self->{added_params}->{-allowempty} = 1;
58:    tie %tmp_cfg, 'Config::IniFiles', %{$self->{added_params}} or $self->_parse_error();

```